### PR TITLE
Replace systemd-mount with pyudev.

### PR DESCRIPTION
**Pull request by MichalHe**
Replace `systemd-mound --list` with _pyudev_.

The current implementation of the _StorageScanner_ actor relies on calling `systemd-mount --list` and parsing its output. Since the command output is not provided in a format that is easy to parse (the data is formatted as a pretty table padded with whitespace) it is error prone to parse it by hand.

This PR replaces the call to `system-mount --list` with the usage of the _pyudev_ library, obtaining the very same information in the same way as  `system-mount --list` does. 

Current status:
- [x] fix unit tests relying on mocking `systemd-mount` command outputs
- [x] manually verify that the messages produced by the old implementation and the new one are the same

### CODE REVIEW

**Review by Rezney — CHANGES_REQUESTED** [review]
Apart from the small typo LGTM. Message in the DB seems to be correct:
```
"systemdmount": [
    {"fs_type": "xfs", "label": "n/a", "model": "n/a", "node": "/dev/vda1", "path": "pci-0000:00:02.0", "uuid": "7d8635ff-fb7c-4e9f-a1ef-13ad8de15b9b", "wwn": "n/a"}, 
    {"fs_type": "xfs", "label": "n/a", "model": "n/a", "node": "/dev/dm-0", "path": "n/a", "uuid": "f3f6c711-f75f-4bd6-af5a-78d590e7fd7d", "wwn": "n/a"}]
```

File: repos/system_upgrade/common/actors/storagescanner/libraries/storagescanner.py [hunk 1]
```diff
@@ -219,22 +220,26 @@ def _get_lvdisplay_info():
...
-        node, path, model, wwn, fs_type, label, uuid = [x for x in entry if x != '']
-        if node == "NODE":
-            # First row of the "systemd-mount --list" output is a header.
-            # Just skip it.
-            continue
+    """
+    Collect the same storage info as provided by the systemd-mount command.
+
+    The actual implementation no longer relies on calling the systemd-mount, but rather collects the same information
+    from udev directly using pyudev. The systemd-mount output parsing has proved not to be unreliable due to
```
[the remark this answers is not in the document]
Comment (pirat89) [reply repos/system_upgrade/common/actors/storagescanner/libraries/storagescanner.py]: done as part of rebasing (fyi @MichalHe )

**Review by pirat89 — APPROVED** [review]

**Review by Rezney — APPROVED** [review]

**Comment by pirat89** [comment]
@MichalHe pls one new rebase

**Review by pirat89 — DISMISSED** [review]
thanks :)

**Comment by pirat89** [comment]
@MichalHe pls one more rebase. we are going to finish this one soon.

**Comment by fernflower** [comment]
leapp-ci build

**Review by pirat89 — APPROVED** [review]

**Pull request merged**